mm2src/mm2_main/tests/mm2_tests/mod.rs

```diff
@@ -10,7 +10,34 @@ mod tendermint_ibc_asset_tests;
 mod tendermint_tests;
 mod z_coin_tests;
 
-// dummy test helping IDE to recognize this as test module
-#[test]
-#[allow(clippy::assertions_on_constants)]
-fn dummy() { assert!(true) }
```

Comment:
same here relating to not recognizing the below mod as a test mod in the IDE https://github.com/KomodoPlatform/atomicDEX-API/blob/42c84d0ec92d74b4a4085ee5fe62b4f2360a56f6/mm2src/mm2_main/tests/mm2_tests_main.rs#L4

Reply:
![image](https://user-images.githubusercontent.com/52405288/233567615-f70747eb-cb07-4932-a7ed-b92da47f94b7.png)
hmm my CLion version has the same problem

Reply:
Sure, thank you